src/components/SwagLandingCard.vue

```diff
@@ -24,14 +29,27 @@
     margin: 0;
     line-height: 133%;
   }
+.flex {
```

Comment:
Avoid overriding of globally available classes. You can reuse classes `flex flex-col items-center` or select a name for the new class and use `@apply directive` like so:
```css
.my-flex {
  @apply flex flex-col items-center;
}
```